server/server.go

```diff
@@ -763,3 +796,59 @@ func (s *Server) Id() string {
 	defer s.mu.Unlock()
 	return s.info.ID
 }
+
+// Event subjects
+const (
+	clientConnectEventSubj     = "_SYS.CLIENT.CONNECT"
+	clientDisconnectEventSubj  = "_SYS.CLIENT.DISCONNECT"
+)
+
+func (s *Server) sendEventNotification(subject string, e interface{}) {
```

Comment:
Should this take bytes payload? And we can handle payload marshaling somewhere else?